models/sales/hubspot__deal_stages.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_sales_enabled','hubspot_deal_enabled'])) }}
```

Comment:
Thought here ---

Why did you choose this approach, rather than joining the new deal_stage model to the final `hubspot__deals` model? Seems like we're doing a lot of duplicative work in this model, but I may be missing something.

Reply:
You're right this is duplicative work. The only difference is the inclusion of the `pipeline.is_active` which is not in the `hubspot__deals` model. I will add that field to the `hubspot__deals` model to make this a cleaner model and remove duplicative code. Thanks!